cfgov/jinja2/v1/ccdb-complaint/ccdb-search.html

```diff
@@ -18,7 +18,19 @@
 {%- endblock %}
 
 {% block content_main %}
-  <div id="ccdb-ui-root"></div>
+  <div id="ccdb-ui-root">
+    <div style="max-width:75%;margin: 6em auto;" class="js-is-hidden">
```

Comment:
You could also do `<div class="js-is-hidden wrapper block block__padded-top block__padded-bottom">` to avoid the inline styles. It'll align with the text, not the menu selection bar, which is how we usually align page content.

Reply:
I tried that first but it seemed too tight visually and there was white space below the footer which seemed somewhat broken
<img width="1531" alt="Screen Shot 2022-04-12 at 8 20 44 AM" src="https://user-images.githubusercontent.com/1558033/162961402-558d0a02-eeb2-44a2-9def-1e9cd225ab5a.png">

Reply:
The wrapper class makes sense though, added in bcf823dcd09b4918328d7ab1d689ce8830ba9c72

Reply:
Weird I don't see that gap at the bottom. Do you see what causes that? 

You could also do it all in classes with `js-is-hidden wrapper u-mt60 u-mb60`, though yes not quite as big a gap (but the same as above "About Us"

<img width="1603" alt="Screen Shot 2022-04-12 at 11 18 50 AM" src="https://user-images.githubusercontent.com/704760/162996310-6f47edcf-b321-44b9-87cd-71a213b8f980.png">

Reply:
Fixed in 34d1b2035b746331c25062a493158a5bb8ea3e48